### DISCUSSION TITLE
增加BufferSize会改善效果吗，还是最好将其设置为0并使用CPU？

### DISCUSSION THREAD

**Discussion by susugo** [XTLS/Xray-core#3542] [Q&A]
我有一台vps他的配置是 2C 1G的，它的cpu很空闲内存总是比较高，我想了解一下如果将BufferSize改为0会降低使用效果么，我自己试了一下没太大感觉，从技术角度这个应该是那个更好呢

**Comment by mmmray** [comment]
The BufferSize where?

**Reply by xqzr** [reply]
https://xtls.github.io/config/policy.html#levelpolicyobject
